components/src/core/InfoListItem/InfoListItem.test.tsx

```diff
@@ -61,12 +59,6 @@ describe('InfoListItem', () => {
     });
  */
 
-    it('renders with avatar', () => {
```

Comment:
No love for the avatar test?

Reply:
Yup, bc now the thing renders a transparent avatar background when avatar = false